### PR TITLE
Add video tag

### PR DESCRIPTION
**Pull request by lai0n**

### CODE REVIEW

File: src/pell.js [hunk 1]
```diff
@@ -22,6 +22,15 @@ const image = () => {
   if (url) execute('insertImage', ensureHTTP(url))
 }
 
+const video = () => {
+  const url = window.prompt('Enter the video URL')
+  const type = window.prompt('Enter the video type')
```
Comment (sylvainpolletvillard) [inline src/pell.js]: If only one source is specified, the `<source>` element is not necessary. You can simply use `<video src="">` and omit the type

**Comment by jaredreich** [comment]
Thanks for the PR! With the latest release, this can be done with a custom action, and therefore shouldn't be part of the core lib. 😄

**Comment by lai0n** [comment]
This 'custom action' is more user-friendly than a video icon?

**Comment by jaredreich** [comment]
The custom action can be a video icon.

E.g.

```js
pell.init({
...
actions: [
    'bold',
    {
      name: 'video',
      icon: '&#128249;',
      title: 'Video',
      result: () => {
        const url = window.prompt('Enter the video URL')
        if (url) pell.exec('insertHTML', `<video src="ensureHTTP(url)" width="400" controls>` + 
        'Your browser does not support HTML5 video.' +
        '</video>')
      }
    },
    'underline'
  ],
...
})
```

**Pull request closed without merging**